libr/core/rvc.c

```diff
@@ -730,6 +732,47 @@ R_API bool r_vc_commit(const char *rp, const char *message, const char *author,
 		eprintf ("Can't commit");
 		return false;
 	}
+	if (!message) {
+#if __WINDOWS__ //fuck you
+	char *whoami = r_sys_whoami ();
+	if (!whoami) {
+		return false;
+	}
+	char *path = r_str_newf ("c:\\\\users\\%s\\appdata\\local\\tmp\\rvcmessage",
+		whoami);
+	free (whoami);
+#else
+	char *path = r_str_newf ("/tmp/rvcmessage");
+#endif
+	if (!path) {
+		return false;
+	}
+	if (!r_file_touch (path)) {
```

Comment:
Try to reduce the amount of return statements by being positive in conditionals and nest them